### PR TITLE
LDP-2328: add optional moderation state to the function cloneNodeByTitle

### PR DESCRIPTION
**Pull request by sindurigf**

### CODE REVIEW

File: tests/helpers/drupal-commands.js [hunk 1]
```diff
@@ -5,15 +5,14 @@ function drush(command) {
...
-   * @param {string} langcode Optional language code prefix for the URL.
+   * @param  {Array<{page: Page, node_title: String, langcode: String}>} array
+   *   Page object, node title and optional language code prefix for the URL.
    *   Note the node title must be in the original language. The visited page
    *   will edit the node's translation if it exists, otherwise the original
    *   language node (just in another UI language).
    * @return {Response} The response.
    */
-  visitNodeEditPage: async ([page, node_title], langcode = '') => {
+  visitNodeEditPage: async ([page, node_title, langcode = '']) => {
```
Comment (rmuit) [inline tests/helpers/drupal-commands.js:15]: Testing @sindurigf 's change made me realize that I don't know Javascript _(well I knew that part)_ and that it is fine to have an optional part of an array, _inside_ an array parameter.

In other words, this change is possible and does not change anything - and that means the recent addition that I did of langcode _outside of the array_ is unnecessary... and inconsistent.

So I 'fixed' that in this same PR.

Comment (rmuit) [reply tests/helpers/drupal-commands.js:15]: None of the current `visitNodeEditPage()` calls pass the optional `langcode`, so nothing breaks. (This was added for LDP-1787 which isn't merged yet.)

**Review by petarbasic — APPROVED** [review]

File: src/Drush/Commands/PlaywrightDrushCommands.php [hunk 2]
```diff
@@ -36,7 +38,7 @@ class PlaywrightDrushCommands extends DrushCommands {
    * @command test:node-clone
    * @aliases nc
    */
-  public function cloneNodeByTitle($node_type, $node_title, $new_node_title) {
+  public function cloneNodeByTitle($node_type, $node_title, $new_node_title, $moderation_state) {
```
Comment (petarbasic) [inline src/Drush/Commands/PlaywrightDrushCommands.php]: I would add a default value for moderation state here as well, even though we have it in the helper. Not a biggie

Comment (rmuit) [reply src/Drush/Commands/PlaywrightDrushCommands.php]: done

**Review by petarbasic — APPROVED** [review]

**Pull request merged**